guake/terminal.py

```diff
@@ -550,12 +554,13 @@ def spawn_sync_pid(self, directory):
             Vte.PtyFlags.DEFAULT,
             directory,
             argv,
-            [f"GUAKE_TAB_UUID={self.uuid}"],
-            GLib.SpawnFlags.DO_NOT_REAP_CHILD,
+            self.envv,
```

Comment:
A better place is to maintain envv at this point each time we need to spawn new termianl.

Reply:
That was my first draft, but then I noticed that the value of self.envv was the same every time so I moved the generation up to __init__ to do it once.